packages/lab/src/form-field-next/FormFieldHelperText.tsx

```diff
@@ -0,0 +1,31 @@
+import clsx from "clsx";
+import { Label, LabelProps, makePrefixer } from "@salt-ds/core";
+import { FormFieldProps } from "./FormFieldNext";
+
+import "./FormFieldHelperText.css";
+
+const withBaseName = makePrefixer("saltFormFieldHelperText");
+
+export interface FormFieldHelperTextProps
+  extends Omit<LabelProps, "variant" | "styleAs"> {
+  helperText: FormFieldProps["helperText"];
+  disabled?: boolean;
+}
+
+export const FormFieldHelperText = ({
+  className,
+  disabled,
+  helperText,
+  ...restProps
+}: FormFieldHelperTextProps) => {
+  return (
+    <Label
```

Comment:
I'm thinking that help text shouldn't return a label. Having two label elements is confusing. Shall we use a 'div' instead? 
Text returns a div element by default. Should be something like:
```
<Text styleAs="label" />
```

Reply:
Zarko suggested the same, have changed :)